src/main/java/org/spongepowered/api/event/cause/entity/damage/source/common/AbstractDamageSourceBuilder.java

```diff
@@ -39,9 +39,9 @@
     protected boolean absolute = false;
     protected boolean magical = false;
     protected boolean creative = false;
+    protected double exhaustion = 0.1;
```

Comment:
...and then I find the impl in the API.

What about making this a nullable `Double`, which indicates “follow vanilla”? Then, when building, this would indicate “do as Vanilla does”.

Just a thought, quite happy to be told this isn’t viable.

Reply:
I like your suggestion, I really couldn't decide the best way to handle this. I would be happy to make this change tomorrow.